packages/metadata/test/providers/cachingProxy.provider.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { afterEach } from "node:test";
 import { beforeEach, describe, expect, it, vi } from "vitest";
```

Comment:
this should be imported from vitest
```suggestion
import { afterEach, beforeEach, describe, expect, it, vi } from "vitest";
```